site/public/js/simple-grading.js

```diff
@@ -708,7 +720,9 @@ function setupSimpleGrading(action) {
                         // Check if cell needs to move to next section
                         new_selector_array[1] += 1;
                         new_selector_array[2] = 0;
-                        new_selector_array[3] = 0;
+
+                        // Check if cell needs to move to null section
+                        if (tries == 1 && new_selector_array[1] != "") {new_selector_array[1] = "";}
```

Comment:
Same thing here about `===` and `!==`
```suggestion
                        if (tries= == 1 && new_selector_array[1] !== "") {new_selector_array[1] = "";}
```

Reply:
additionally, it would be more readable to format this such that it isn't on one line.